assets/js/base/components/cart-checkout/product-sale-badge/index.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { formatPrice } from '@woocommerce/base-utils';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+/**
+ * Returns a low stock badge for a line item.
```

Comment:
Comment pasteo :)

Reply:
Removed in 4a60a67b57a8d8d5f40fc7467b79c69c973fd942.